core/src/test/java/com/linecorp/armeria/internal/annotation/AnnotatedHttpServiceFactoryTest.java

```diff
@@ -224,4 +259,31 @@ public String userDefinedRepeatableDecorator() {
             return "";
         }
     }
+
+    @PathPrefix(HOME_PATH_PREFIX)
+    static class PathPrefixServiceObject {
+
+        @Get("/hello")
```

Comment:
Could you add test cases for verifying exception with `@Get`, `@Post`? 😀

Reply:
Yup I will do that. Thanks for suggestion.